dist/obs-server.spec

```diff
@@ -467,16 +473,28 @@ touch /srv/www/obs/api/log/production.log
 chown %{apache_user}:%{apache_group} /srv/www/obs/api/log/production.log
 
 %restart_on_update memcached
+%service_add_post %{obs_api_support_scripts}
 # We need to touch the last_deploy file in the post hook
 # to update the timestamp which we use to display the
 # last deployment time in the API
 touch /srv/www/obs/api/last_deploy || true
 
+# Upgrading from SysV obsapidelayed.service to systemd obs-api-support.target
+if [ "$1" -gt 1 ]; then
+  if /bin/systemctl --quiet is-enabled obsapidelayed.service; then
+    /bin/systemctl enable obs-api-support.target
+  fi
+  if /bin/systemctl --quiet is-active obsapidelayed.service; then
```

Comment:
[nitpick] /bin/systemctl is obsolete and just a fallback symlink. systemctl is in /usr/bin (and there is no real need to call it here with absolute path)

Reply:
`/bin/` removed.